src/lib/Containers/Conversation.tsx

```diff
@@ -140,9 +136,8 @@ export default Relay.createContainer(Conversation, {
               }
               edges {
                 node {
-                  is_from_user
-                  raw_text
-                  created_at
+                  id
```

Comment:
wow, ok - I'm guessing this is what I was missing to get the `fragment` stuff working.